src/publishMain.ml

```diff
@@ -817,7 +817,9 @@ let () =
   | OpamStd.Sys.Exit i -> exit i
   | Failure e ->
     OpamConsole.error "%s" e;
+    OpamConsole.msg "%s" (Printexc.get_backtrace ());
```

Comment:
The backtrace has a strong chance of being trashed in `OpamConsole.error`
```suggestion
    let bt = Printexc.get_backtrace () in
    OpamConsole.error "%s" e;
    OpamConsole.msg "%s" bt;
```
worth combining with a `Printexc.record_backtrace true;` call at L803

Reply:
ah indeed. Fixed.